django/santropolFeast/member/models.py

```diff
@@ -209,3 +209,20 @@ class Meta:
         related_name='allergic_clients',
         blank=True
     )
+
+
+class Referent (models.Model):
```

Comment:
Be carefull on the name of the Class. Here i think you describe a reference and not a referent.